documentation/prettier/readme.md

```diff
@@ -0,0 +1,57 @@
+# Prettier Config Documentation
+
+> Documentation for the `@strv/prettier-config` module.
+
+## Usage
+
+### Installing
+
+Run:
```

Comment:
Cosmetic nitpick, but I'd prefer to drop the `Installing` heading and _Run:_ text completely and just keep the contents directly under `Usage` header.